### PR TITLE
docs: Remove miscreant line in argo.md

### PR DESCRIPTION
**Pull request by max-sixty**
Checklist:

Is this line supposed to be here? It's causing the docs to look like this:

image

### CODE REVIEW

**Review by alexec — CHANGES_REQUESTED** [review]
Good spot.

argo.md is generated from the source code, you need to fix this line in cmd/argo/commands/cron/root.go:

```golang
Short: "manage cron workflows\n\nNextScheduledRun assumes that the workflow-controller uses UTC as its timezone",
````

You also need to sign-off your commits to pass the DCO check, and fix the PR tile to pass the Semantic Commit check.

**Comment by max-sixty** [comment]
Any thoughts on why "docs: Remove miscreant line in argo.md" doesn't pass the semantic check while  "docs: Expand on controller security" does?

File: cmd/argo/commands/cron/root.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 func NewCronWorkflowCommand() *cobra.Command {
 	var command = &cobra.Command{
 		Use:   "cron",
-		Short: "manage cron workflows\n\nNextScheduledRun assumes that the workflow-controller uses UTC as its timezone",
```
Comment (alexec) [inline cmd/argo/commands/cron/root.go:10]: I think you actually want to do this:

```golang
Short: 'Short: "manage cron workflows'
Long: `manage cron workflows

NextScheduledRun assumes that the workflow-controller uses UTC as its timezone
`
```

Comment (max-sixty) [reply cmd/argo/commands/cron/root.go:10]: Ah thanks. My first go PR.

Comment (max-sixty) [reply cmd/argo/commands/cron/root.go:10]: Copying this: https://github.com/argoproj/argo/blob/master/cmd/argo/commands/root.go#L27-L28

I haven't included the comment again in the Long field

**Comment by max-sixty** [comment]
Do I need to rebuild the docs locally? I didn't see any instructions in Contributing.md

**Comment by max-sixty** [comment]
Still not sure what's causing the error — do I need to rebuild the docs locally and commit?

**Comment by max-sixty** [comment]
Still keen to contribute and fix this, but I do need guidance on why the docs build is failing. Let me know if I'm missing something obvious...

**Comment by alexec** [comment]
Three options to fix the `codegen`:

1. Run 'make codegen' and commit changes. If there are no changes, try using `make codegen -B` which forces code generation.
2. Run `go run ./hack/cli`(which is the step that generates the code).
3. Copy the patch printed in the job and apply using git apply, e.g. `pbpaste|cut -c 30-|git apply`. You must get ever line from `git diff` to the last line of code changed, which can be tricky to get right.

**Comment by max-sixty** [comment]
Great, done, thanks!

**Review by alexec — APPROVED** [review]
LGTM

**Pull request merged**